src/Http/Controllers/Auth/SocialiteController.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Controllers\Auth;
+
+use Blumilk\Meetup\Core\Http\Controllers\Controller;
+use Blumilk\Meetup\Core\Models\User;
+use Laravel\Socialite\Facades\Socialite;
+
+class SocialiteController extends Controller
+{
+    public function redirectToGoogle()
+    {
+        return Socialite::driver("google")->redirect();
+    }
```

Comment:
```suggestion
    }
    
```
and below the same.

Reply:
Are you using `ECS`  for code styling?

Reply:
Yes, I am but it always delete empty line at the end of file

Reply:
Do you use Blumilk ECS?

Reply:
I haven't changed anything in the configuration and I'm using `composer ecs` command from composer.json